pkg/ddc/jindo/worker.go

```diff
@@ -214,3 +224,24 @@ func (e *JindoEngine) CheckWorkersReady() (ready bool, err error) {
 
 	return
 }
+
+// getWorkerSelectors gets the selector of the worker
+func (e *JindoEngine) getWorkerSelectors() string {
+	labels := map[string]string{
+		"release":     e.name,
+		POD_ROLE_TYPE: WOKRER_POD_ROLE,
+		"app":         common.JINDO_RUNTIME,
+	}
+	labelSelector := &metav1.LabelSelector{
+		MatchLabels: labels,
+	}
+
+	selectorValue := ""
+	selector, err := metav1.LabelSelectorAsSelector(labelSelector)
+	if err != nil {
+		e.Log.Error(err, "Failed to parse the labelSelector of the runtime", "labels", labels)
+	} else {
+		selectorValue = selector.String()
+	}
+	return selectorValue
```

Comment:
Why not just use `labels.Selector` instead of converting it to a string ?

Reply:
the selector get by LabelSelectorAsSelector, not the label's property